server/routes/reviewer.ts

```diff
@@ -81,3 +67,23 @@ export = function CreateReviewRoutes({ offenderService, reportDetailBuilder, rev
     },
   }
 }
+
+function viewIncidents(page, tab, reviewService) {
```

Comment:
I wouldn't try to avoid the duplication by keeping the 2 handler separate - the pages are considerably different. 
Don't need to support search on incomplete reports at all which leads to a very simple handler function.

Reply:
ok